app.js

```diff
@@ -1,10 +1,27 @@
+/*jslint node: true */  
 'use strict';
 
 var ari = require('ari-client');
 var util = require('util');
 var sla = require('./lib/sla.js');
 
-ari.connect('http://127.0.0.1:8088', 'ariUser', 'boogers', clientLoaded);
+ari.connect('http://127.0.0.1:8088', 'user', 'pass', clientLoaded);
```

Comment:
You will eventually want to move the URL/user/pass of the connect method into a config file loaded by this file.